src/main/java/com/epam/preprod/sirenko/containers/CommandContainer.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.preprod.sirenko.containers;
+
+import com.epam.preprod.sirenko.command.Command;
+import com.epam.preprod.sirenko.command.ExitProgramCommand;
+import com.epam.preprod.sirenko.command.GetProductsListCommand;
+import com.epam.preprod.sirenko.command.NoCommandCommand;
+import com.epam.preprod.sirenko.dao.ProductDAO;
+import com.epam.preprod.sirenko.services.GetAllProductsListService;
+import com.epam.preprod.sirenko.services.ProductService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * All commands holder
+ */
+public class CommandContainer {
+	
+	private Map<Integer, Command> commands = new HashMap<>();
```

Comment:
Maybe we can just use String as a key to not convert scanner input to Integer every time.

Reply:
done